toolchain/lex/lex.cpp

```diff
@@ -140,9 +186,14 @@ class [[clang::internal_linkage]] Lexer {
   auto LexSymbolToken(llvm::StringRef source_text, ssize_t& position)
       -> LexResult;
 
+  template <TokenKind::RawEnumType Kind>
+  auto LexTypeLiteralToken(llvm::StringRef suffix, int32_t byte_offset)
```

Comment:
I think it would be helpful for this function to have a comment, especially to clarify how this differs from `LexWordAsTypeLiteralToken`

Reply:
Removed this function now that I'm not excessively trying to track the kind as a type here.